iron-ajax.html

```diff
@@ -291,7 +291,14 @@
     },
 
     get requestHeaders() {
-      var headers = Object.create(this.headers || {});
+      var headers = {},
+          header;
+
+      if(this.headers instanceof Object) {
```

Comment:
After [discussing](https://github.com/PolymerElements/iron-ajax/pull/6#discussion-diff-29492065) that object cloning was the intended behavior, the changing commit was removed.

`this.headers` should be verified to be an Object, because if a string was passed in, it would be sliced as Object of single characters and each of them sent as request header. (Unit Test added)